### PR TITLE
Renaming servicebox to primaza

### PR DESCRIPTION
**Pull request by cmoulliard**
- Renaming servicebox to primaza
- Changing the pom parent name and artifactId like for the module

### CODE REVIEW

File: .github/install_service_box.sh [hunk 1]
```diff
@@ -18,11 +18,11 @@ mvn clean install -DskipTests -Ppush-images,kubernetes -Dquarkus.container-image
...
+helm install --dependency-update primaza-app primaza-app/target/helm/kubernetes/primaza-app -n $KUBERNETES_NAMESPACE --set app.image=$KIND_REGISTRY/$KIND_REGISTRY_GROUP/primaza-app:$VERSION
+kubectl wait --for=condition=ready --timeout=5m pod -l app.kubernetes.io/name=primaza-app -n $KUBERNETES_NAMESPACE
 
-POD_NAME=$(kubectl get pod -l app.kubernetes.io/name=servicebox-app -n $KUBERNETES_NAMESPACE -o name)
-RESULT=$(kubectl exec -i $POD_NAME --container servicebox-app -n $KUBERNETES_NAMESPACE -- sh -c "curl -s -i localhost:8080/home")
+POD_NAME=$(kubectl get pod -l app.kubernetes.io/name=primaza-app -n $KUBERNETES_NAMESPACE -o name)
+RESULT=$(kubectl exec -i $POD_NAME --container primaza-app -n $KUBERNETES_NAMESPACE -- sh -c "curl -s -i localhost:8080/home")
 if [[ "$RESULT" = *"500 Internal Server Error"* ]]
 then
   echo "Service Box is not working"
```
Comment (aureamunoz) [inline .github/install_service_box.sh:28]: There are plenty of `Service Box`remainings in the `echo` commands

**Review by aureamunoz — CHANGES_REQUESTED** [review]
There are plenty of Service Box remainings in the echo commands in the .github folder

File: .github/install_service_box.sh (hunk 1 above)
Comment (cmoulliard) [reply .github/install_service_box.sh:28]: PR updated ;-)

**Comment by cmoulliard** [comment]
I think that we are good @aureamunoz

**Review by aureamunoz — APPROVED** [review]

**Pull request merged**